Funcky.Analyzers/Funcky.Analyzers/CompilationExtensions.cs

```diff
@@ -14,5 +14,9 @@ public static class CompilationExtensions
 
     public static INamedTypeSymbol? GetOptionType(this Compilation compilation) => compilation.GetTypeByMetadataName("Funcky.Monads.Option");
 
+    public static INamedTypeSymbol? GetOptionExtensionsType(this Compilation compilation) => compilation.GetTypeByMetadataName("Funcky.Monads.OptionExtensions");
+
     public static INamedTypeSymbol? GetSequenceType(this Compilation compilation) => compilation.GetTypeByMetadataName("Funcky.Sequence");
+
+    public static INamedTypeSymbol? GetFunctionalType(this Compilation compilation) => compilation.GetTypeByMetadataName("Funcky.Functional");
```

Comment:
Vielleicht die Namen in FunckyWellKnownNames extrahieren?

Reply:
Fixed in [8e1d285](https://github.com/polyadic/funcky/pull/680/commits/8e1d28538a1e5b960ba8dc9fd3a227bb34da54f7)